apps/api/v2/src/modules/webhooks/controllers/event-types-webhooks.controller.e2e-spec.ts

```diff
@@ -0,0 +1,295 @@
+            eventTriggers: ["BOOKING_CREATED", "BOOKING_RESCHEDULED", "BOOKING_CANCELLED"],
+            active: false,
+            payloadTemplate: "string",
+            eventTypeId: eventType.id,
+          },
+        } satisfies WebhookOutputResponseDto);
+      });
+  });
+
+  it("/webhooks/event-types/:eventTypeId/webhooks/:webhookId (GET) should fail to get a webhook that does not exist", () => {
+    return request(app.getHttpServer()).get(`/v2/event-types/${eventType.id}/webhooks/90284`).expect(404);
+  });
+
+  it("/webhooks/event-types/:eventTypeId/webhooks/:webhookId (GET) should fail to get a webhook of an eventType that does not belong to user", () => {
+    return request(app.getHttpServer())
+      .get(`/v2/event-types/${otherEventType.id}/webhooks/${otherWebhook.id}`)
+      .expect(403);
+  });
+
+  it("/webhooks/event-types/:eventTypeId/webhooks/:webhookId (GET) should fail to get a webhook that does not belong to the eventType", () => {
+    return request(app.getHttpServer())
+      .get(`/v2/event-types/${eventType.id}/webhooks/${otherWebhook.id}`)
+      .expect(400);
+  });
+
+  it("/webhooks (GET)", () => {
+    return request(app.getHttpServer())
+      .get(`/v2/event-types/${eventType.id}/webhooks`)
+      .expect(200)
+      .then((res) => {
+        const responseBody = res.body as WebhooksOutputResponseDto;
+        responseBody.data.forEach((webhook) => {
+          expect(webhook.eventTypeId).toBe(eventType.id);
+        });
+      });
+  });
+
+  it("/webhooks (GET)", () => {
+    return request(app.getHttpServer())
+      .get(`/v2/event-types/${eventType2.id}/webhooks`)
+      .expect(200)
+      .then((res) => {
+        const responseBody = res.body as WebhooksOutputResponseDto;
+        responseBody.data.forEach((webhook) => {
+          expect(webhook.eventTypeId).toBe(eventType2.id);
+        });
+      });
+  });
+
+  it("/webhooks/event-types/:eventTypeId/webhooks/:webhookId (DELETE)", () => {
+    return request(app.getHttpServer())
+      .delete(`/v2/event-types/${eventType.id}/webhooks/${webhook.id}`)
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toMatchObject({
+          status: "success",
+          data: {
+            id: expect.any(String),
+            subscriberUrl: "https://example.com",
+            eventTriggers: ["BOOKING_CREATED", "BOOKING_RESCHEDULED", "BOOKING_CANCELLED"],
+            active: false,
+            payloadTemplate: "string",
+            eventTypeId: eventType.id,
+          },
+        } satisfies WebhookOutputResponseDto);
+      });
+  });
+
+  it("/webhooks/event-types (DELETE)", () => {
+    return request(app.getHttpServer())
+      .delete(`/v2/event-types/${eventType2.id}/webhooks`)
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toMatchObject({
+          status: "success",
+          data: "1 webhooks deleted",
+        } satisfies DeleteManyWebhooksOutputResponseDto);
+      });
+  });
+
```

Comment:
This first “webhooks” segment in all of these test cases shouldn’t be there, right?